tests/generation_test.go

```diff
@@ -41,7 +41,10 @@ func TestCore(t *testing.T) {
 func TestValidation(t *testing.T) {
 	t.Parallel()
 
-	testExamples(t, basicConfig, "./data/validation")
+	cfg := basicConfig
+	cfg.StructVerify = true
+
+	testExamples(t, cfg, "./data/validation")
 }
```

Comment:
I would prefer having two suites of tests, one with the classic validation and one with the struct verify. wdyt?